addons/product/report/product_pricelist_report.py

```diff
@@ -55,13 +57,15 @@ def _get_report_data(self, active_model, active_ids, pricelist_id, quantities, r
             'quantities': quantities,
             'is_product_tmpl': is_product_tmpl,
             'is_html_type': report_type == 'html',
+            'is_visible_title': is_visible_title,
```

Comment:
```suggestion
            'is_visible_title': self.env.context.get('is_visible_title'),
```
Can't you move the full specification here, without passing it as parameter ?
My last previous comment was wrong, I didn't notice render_values was the result of a method call, I only saw it as a dict directly specified :man_facepalming:

Reply:
@Feyensv When print action occurs, there will be no context because there is data
see : _get_report_values 
https://github.com/odoo/odoo/blame/master/addons/product/report/product_pricelist_report.py#L11

Reply:
Indeed, the context is only given in some specific cases for report actions :+1: 
I've taken the occasion to clean up a bit the report logic in a separate commit and pass the data in a consistent way, other than the context :).